--- xorg-server-1.7.7/hw/xfree86/fbdevhw/fbdevhw.c	2018-01-11 21:24:00.394610184 +0200
+++ xorg-server-1.7.7/hw/xfree86/fbdevhw/fbdevhw.c.new	2018-01-11 21:24:23.000000000 +0200
@@ -340,7 +340,7 @@
      * a device.  However, on a mixed architecture the base addresses of PCI
      * devices, even for 32-bit applications, may be higher than 0x0f0000000.
      */
-
+/*
     for (i = 0; i < 8; i++) {
 	sprintf(filename,"/dev/fb%d",i);
 	if (-1 == (fd = open(filename,O_RDWR,0))) {
@@ -374,7 +374,7 @@
 	}
 	return fd;
     }
-
+*/
     if (namep)
       *namep = NULL;
 
